### ISSUE TITLE
Phenological stages for SEGI

### ISSUE THREAD

**Issue by ngoj1** [christophe-rd/fuelinex#13]
One thing we have yet to settle is how we describe the budburst (if we can even call it that...) for SEGI (_Sequoiadendron giganteum_). Alongside its sister taxon SESE (_Sequoia sempervirens_) it seems to exhibit dormancy and budburst phenology that is not at all like the angiosperms in this experiment, or even the other conifer PIST (_Pinus strobus_). 

![nonGrowing](https://github.com/christophe-rd/fuelinex/assets/87724921/2bad2a3c-c732-4512-8673-c43295a04576)
![apicalBurst](https://github.com/christophe-rd/fuelinex/assets/87724921/a59e449f-f6a5-452b-b1c9-5df4c6c212c5)
![branchletBud](https://github.com/christophe-rd/fuelinex/assets/87724921/220f017f-ad9c-4351-890f-82400419d4b8)
![branchletBurst](https://github.com/christophe-rd/fuelinex/assets/87724921/0ec7cdcf-7ab1-4a3e-9492-44b6c2b076b1)

The images above demonstrate:
1) Non-growing branchlet tips
2) burst of the apical meristem
3) buds at branchlet tips
4) burst of branchlet meristems

For a long time, many of the SEGI plants looked just like those in (1), with both branchlets and apical meristems not displaying any kind of elongation, colour change, or swelling. Recently however, those trees that have resumed growth display traits from images (2) to (4). It appears that SEGI "buds" only really develop and swell prominently well AFTER winter has passed. The reddish colouration is first retained as the meristems swell, as in (3), then they start to brighten and elongate as in (4). This is also clearly seen in the apical meristem, which has a distinctly brighter green colour than the stem below it.

If we look at SESE bursting at the same time:
![seseBurst](https://github.com/christophe-rd/fuelinex/assets/87724921/89c95645-67ee-4307-8814-7745ea456f41)

We can see a similar pattern here. In the winter before Fredi left, there was no sign at all that the SESE had begun to develop any kind of noticeable bud as with the angiosperms. The only sign that it was entering dormancy was the stagnation of shoot elongation. However, after winter has passed, we can see that large buds at the apex and also along the feather-like branchlets started to form during spring warming, followed by a distinct bursting out of the bright green branchlets. 

While I couldn't find information on the budburst of these two species (or even the sister to this clade, _Metasequoia glyptostroboides_), I did find [this paper by Grosfeld & Barthélémy (2004)](https://doi.org/10.1111/j.1095-8339.2004.00331.x) that argues the cessation of growth in some Cupressaceae (of which SEGI and SESE are members) like _Juniperus_, _Chamaecyparis_, and some Patagonian species is not due to true dormancy but rather just environmental cues limiting growth rate. 

With this in mind, I propose that SEGI (and SESE, if we use it again) should follow this phenological stage pattern:
1) Swelling of the meristem
2) Colour change (difference between old, darker/redder foliage and new, brighter foliage)
3) Elongation of the developing branchlet/shoot, leaves still appressed
4) Unfurling of leaves away from branchlet/shoot

**Comment by lizzieinvancouver** [comment]
@ngoj1 Thanks for your great work on this! The stages sound fine to me and as long as we clearly document each stage with several photos we should be okay. The only last thing I suggest you check is [the BBCH manual](https://www.politicheagricole.it/flex/AppData/WebLive/Agrometeo/MIEPFY800/BBCHengl2001.pdf) and confirm there's nothing similar with additional stages we should note.

**Comment by ngoj1** [comment]
I looked over the section on principal growth stages and it seems like their stage 0 maps well to our stage 1, their 3 to our 3, and their 1 to our 4...I wonder if this is because the BBCH manual discusses angiosperms (and particularly agricultural crops)? @DeirdreLoughnan on that original sheet we used to monitor pheno stages that was based on broadleaf trees, was there a stage 0 mentioned?

**Comment by DeirdreLoughnan** [comment]
@lizzieinvancouver @christophe-rd @ngoj1 Yes, there is a stage zero on the [scale that is being used](https://github.com/christophe-rd/fuelinex/blob/main/methodology/phenological%20stages.jpg). 

I have a slightly different interpretation from your comparison @ngoj1 (although I think it is partially a typo), since there is a zero:  fuelinex 0 = BBCH 0, fuelinex 1 = BBCH 1-3, fuelinex 2 = BBCH 7-9, fuelinex 3 = BBCH 10-11, fuelinex 4 = BBCH 11 +

But @christophe-rd you should also weigh in! It is cool to see how much the segi has changed since I last did the observations!

**Comment by christophe-rd** [comment]
@ngoj1 , thanks so much for this! It's very well detailed! 

A few things:
I would like to confirm that we match the stages and the photos you posted.
Stage 0: Picture 1
Stage 1:
Stage 1:
Stage 1:

**Comment by christophe-rd** [comment]
@ngoj1 , thanks so much for this! It's very well detailed!

First, I agree with your growth stages @ngoj1. However, I think stage 1 and 2 arrive simultaneously. I might be wrong, but I didn't notice that the meristems swell before they changed colour. 

I agree with stages 2 and 3, but if we merge 1-2, then we will have only 3 stages. 

Also, I have a question regarding BBCH. Can we compare the stages from BBCH to sequoia's growth stages since they apply mostly to fruit trees? For instance, I don't think Sequoia follows the sequence  1. leaf development 2. formation of side shoots 3. stem elongation. 
I think @ngoj1 's growth stages sequence is accurate, but I believe comparing to the BBCH's stages might be confusing. I believe Sequoia should have its separate development stages for this experiment. What do you think? 

I will create a git issue for Pinus as well. 

Also, I saw that there is a macro lens with the 5d. @lizzieinvancouver, can I use it to take pictures of the phenostages?

**Comment by christophe-rd** [comment]
@lizzieinvancouver that sounds good. Thanks!
